omod/src/main/resources/config.xml

```diff
@@ -126,34 +131,176 @@
 		<property>${project.parent.artifactId}.openempi.enableProbabilisticMatching</property>
 		<defaultValue>false</defaultValue>
 		<description>
-			Specifies if probably matching should be used
+			Specifies if probability matching should be used (GSOC OpenEMPI Specific Implementation)
 		</description>
 	</globalProperty>
 
 	<globalProperty>
-		<property>${project.parent.artifactId}.mpi.personIdentifierId</property>
-		<defaultValue></defaultValue>
+		<property>${project.parent.artifactId}.mpi.personIdentifierTypeUuid</property>
+		<defaultValue>a5d38e09-efcb-4d91-a526-50ce1ba5011a</defaultValue>
 		<description>
-			Specifies id of identifier type which will be used for Mpi Person Id
+			Specifies UUID of the identifier type that is used as the Global Identifier Domain (GID) in the MPI.
+			It is generated by the MPI and is persisted in OpenMRS.
 		</description>
 	</globalProperty>
-	
+
 	<globalProperty>
-		<property>${project.parent.artifactId}.birthdateEstimationStartMonth</property>
-		<defaultValue>0</defaultValue>
+		<property>${project.parent.artifactId}.local_mpi_identifierTypeMap.OpenEMPI ID</property>
+		<defaultValue>a5d38e09-efcb-4d91-a526-50ce1ba5011a:2.16.840.1.113883.4.357:hl7</defaultValue>
 		<description>
-			The month from which the birth date is estimated, with January having a value of 0
+			Specifies the mapping from OpenEMPI Object ID (OID) to the local OpenMRS Application's identifier for OpenEMPI ID UUID.
+			OpenEMPI ID is the default Global Identifier Domain (GID) for OpenEMPI but this can be changed.
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.local_mpi_identifierTypeMap.OpenMRS ID</property>
+		<defaultValue>05a29f94-c0ed-11e2-94be-8c13b969e334:2.25.71280592878078638113873461180761116318:PI</defaultValue>
+		<description>
+			Specifies the mapping from OpenEMPI Object ID (OID) to the local OpenMRS Application's identifier for OpenMRS ID UUID.
+			OpenMRS ID is the default generated identifier for OpenMRS but this can be changed.
 		</description>
 	</globalProperty>
 
     <globalProperty>
         <property>${project.parent.artifactId}.biometrics.implementation</property>
         <defaultValue></defaultValue>
         <description>
-            Which Biometrics engine implementation to should we connect to? Specify a Spring bean name, or leave blank to disable
+            Which Biometrics engine implementation to should we connect to?
+			Specify a Spring bean name, or leave blank to disable
         </description>
     </globalProperty>
 
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pix.errorHandler.implementation</property>
+		<defaultValue></defaultValue>
```

Comment:
The module comes with a single implementation of some of these classes, why not set them as the defaults

Reply:
The error handler is from Outgoing Message Exceptions module, it was not set so that if that module is not deployed, the code still works just without error handling